### PR TITLE
InfluxDB 2.7

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.7/api-guide/api_intro.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: API Quick Start
+seotitle: Use the InfluxDB API
+description: Interact with InfluxDB using a rich API for writing and querying data and more.
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/api_intro.md:4]:
```suggestion
description: Interact with InfluxDB v2 using a rich API for writing and querying data and more.
```

File: content/influxdb/v2.7/api-guide/api_intro.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+For detailed documentation on the entire API, see [InfluxDBv2 API Reference](/influxdb/v2.7/reference/api/#influxdb-v2-api-documentation).
+
+{{% note %}}
+If you need to use InfluxDB {{< current-version >}} with **InfluxDB 1.x** API clients and integrations, see the [1.x compatibility API](/influxdb/v2.7/reference/api/influxdb-1x/).
+{{% /note %}}
+
+## Bootstrap your application
+
+With most API requests, you'll need to provide a minimum of your InfluxDB URL, Organization, and Authorization Token.
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/api_intro.md:27]:
```suggestion
With most API requests, you'll need to provide a minimum of your InfluxDB URL and Authorization Token (API Token).
```

File: content/influxdb/v2.7/api-guide/client-libraries/browserjs.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+aliases:
+  - /influxdb/v2.7/reference/api/client-libraries/browserjs/
+  - /influxdb/v2.7/api-guide/client-libraries/browserjs/write
+  - /influxdb/v2.7/api-guide/client-libraries/browserjs/query
+related:
+  - /influxdb/v2.7/api-guide/client-libraries/nodejs/write/
+  - /influxdb/v2.7/api-guide/client-libraries/nodejs/query/
+---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to interact with the InfluxDB API in browsers and front-end clients. This library supports both front-end and server-side environments and provides the following distributions:
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/client-libraries/browserjs.md:23]:
```suggestion
Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to interact with the InfluxDB v2 API in browsers and front-end clients. This library supports both front-end and server-side environments and provides the following distributions:
```

File: content/influxdb/v2.7/api-guide/client-libraries/go.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+        map[string]interface{}{"avg": 24.5, "max": 45},
+        time.Now())
+    // Write point immediately
+    writeAPI.WritePoint(context.Background(), p)
+    // Ensures background processes finishes
+    client.Close()
+}
+```
+## Query data from InfluxDB with Go
+Use the Go library to query data to InfluxDB.
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/client-libraries/go.md:123]:
```suggestion
Use the Go library to query data stored in InfluxDB.
```

File: content/influxdb/v2.7/api-guide/client-libraries/go.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+1. Create a Flux query and supply your `bucket` parameter.
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/client-libraries/go.md:125]:
```suggestion
1. Create a Flux query and pass the `bucket` parameter with your bucket name.
```

File: content/influxdb/v2.7/api-guide/client-libraries/nodejs/install.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
+---
+title: Install the InfluxDB JavaScript client library
+seotitle: Install the InfluxDB Node.js JavaScript client library
+description: >
+  Install the JavaScript client library to interact with the InfluxDB API in Node.js.
```
Comment (jstirnaman) [inline content/influxdb/v2.7/api-guide/client-libraries/nodejs/install.md:5]:
```suggestion
  Install the Node.js JavaScript client library to interact with the InfluxDB v2 API.
```

File: content/influxdb/v2.7/influxdb-templates/create.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+
+- [Create a template](#create-a-template)
+- [Export resources to a template](#export-resources-to-a-template)
+- [Include user-definable resource names](#include-user-definable-resource-names)
+- [Troubleshoot template results and permissions](#troubleshoot-template-results-and-permissions)
+- [Share your InfluxDB templates](#share-your-influxdb-templates)
+
+## Create a template
+
+Creating a new organization to contain only your template resources is an easy way
```
Comment (jstirnaman) [inline content/influxdb/v2.7/influxdb-templates/create.md:32]:
```suggestion
Creating an organization to contain only your template resources is an easy way
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
A few small fixes.

**Comment by sanderson** [comment]
Thanks for the review @jstirnaman!

**Pull request merged**